groot/templates/admin/group_permissions.html

```diff
@@ -1,94 +1,97 @@
-{% extends "admin/base_site.html" %}
-{% load i18n l10n admin_urls static %}
+{% extends 'admin/base_site.html' %}
+{% load admin_urls %}
```

Comment:
Ready to go it might worth adding extra blank space above so we split not common things in diff groups for readability.